cmd/main.go

```diff
@@ -110,12 +114,37 @@ func main() {
 	}
 
 	nodesComDetails := []types.ComDetails{}
+
+	err = debug.CreateNamespace(cs, consts.DefaultDebugNamespace)
+	if err != nil {
+		panic(err)
+	}
+	defer func() {
+		output, err := exec.Command("oc", "delete", "ns", consts.DefaultDebugNamespace).CombinedOutput()
```

Comment:
Should we use the api instead of the exec command? Can be debug.DeleteNamespace

Reply:
fixed, added the new function